defaults/main.yml

```diff
@@ -10,7 +10,17 @@ nginx_group_allow: []
 nginx_host_allow: []
 
 # List of Debian packages installed by this role
-nginx_base_packages: [ 'nginx-full' ]
+nginx_base_packages: [ 'python-passlib' ]
+nginx_flavor: 'full'
+
+# The passenger version is very fragile until Phusion makes a change or two
+nginx_flavor_map:
+  'full': [ 'nginx-full' ]
+  'passenger': [ 'nginx-extras=1:1.6.2-2.4.0.59~wheezy1', 'passenger' ]
+
+# Passenger configuration
+passenger_max_pool_size: 30
+passenger_options: False
```

Comment:
These should be namespaced as `nginx_passenger_`, remember to also change them in other places.